### PR TITLE
How to recieve emails with emailjs in vue3

### PR DESCRIPTION
**Pull request by mercybassey**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes

`This closes #issue-number`
#5580

### CODE REVIEW

File: content/articles/how-to-recieve-emails-using-emailjs-in-vue3/index.md [hunk 1]
```diff
@@ -0,0 +1,382 @@
+# How to recieve emails using Emailjs in Vue3
+
+## **Introduction**
```
Comment (jumamiller) [inline content/articles/how-to-recieve-emails-using-emailjs-in-vue3/index.md:3]: Hi, just to ensure your article is well polished, remove the spacings between the heading and the first sentence. This should be done across the entire article.

. Ensure you use the correct headers i.e use level 3 (###) and not ##
. Remove the bold feature on the header i.e, instead of  '**Introduction**', use 'Introduction'.

> Ensure these edits are done on the entire article!

**Review by jumamiller — COMMENTED** [review]
Hi, I recently started learning Vue3 and would love to review this article.

However, I have noted a few areas of your article that needs improvements:

Do you think adding styling (CSS) to this tutorial is necessary, or do we only focus on the core objective? Kindly let me know what you think!
- Now assume I'm a newbie to Vue. Are the code snippets well explained to the level I won't have any problems following? 
- To comment on the above, I started learning Vue 2 and later realized that it's somehow different from Vue 3. So I had to quit and pick the current version (Vue3). At some point, I have seen you refer readers to Vue 2; what will happen if I have no clue about Vue 2?

> Feel free to leave me a comment here on what you think. Have a great new year ahead!

**Comment by srishilesh** [comment]
@jumamiller 

Make sure to add necessary labels that allows the CMs to keep track on the statuses of the PRs.

Assigning this to you.

**Comment by jumamiller** [comment]
Hi @srishilesh, I already have a couple of articles in the queue, I will jump onto this one once my queue is cleared. Thank you!

**Comment by Kayere** [comment]
Plagiarism check done ✅

**Comment by mercybassey** [comment]
> 
Firstly, I personally have an eye for a good UI so  I  happen to style, every single HTML tag I write. So for the purpose of the article,  I feel it is necessary. That's one thing we Frontend developers cannot do without.

Secondly, I  figured that since this article Is meant for only Vue3, there will be no need to make references to Vue2. I'd like to know if i can still go ahead with the corrections you asked to be made.

**Comment by jumamiller** [comment]
> > 
> 
> @jumamiller Firstly, I personally have an eye for a good UI so I happen to style, every single HTML tag I write. So for the purpose of the article, I feel it is necessary. That's one thing we Frontend developers cannot do without.
> 
> Secondly, I figured that since this article Is meant for only Vue3, there will be no need to make references to Vue2. I'd like to know if I can still go ahead with the corrections you asked to be made.

Great response! I'll proceed with the review!

**Comment by jumamiller** [comment]
Hi, just on a quick glance, you have a couple of grammatical mistakes. Kindly ensure that your article is passed on to Grammarly and well polished!

**Comment by mercybassey** [comment]
@jumamiller I have made the grammatical corrections as requested.

**Comment by mercybassey** [comment]
@jumamiller  I removed the references to vue2, as it is not necessary to check out an old version of a language/framework before one can learn the new version. I'd like to know if I do need to make other corrections.

**Comment by jumamiller** [comment]
Hello @mercybassey, you have a couple of grammatical mistakes, in this case I mean, for example, you have misspelled words like `recieve` instead of `receive`. Kindly use[ Grammarly](https://app.grammarly.com) to help you fix these minor mistakes!

FYI: Any further revision may lead to article closure which I feel is avoidable in this case if you use the right tools.

**Comment by mercybassey** [comment]
@jumamiller Done

**Comment by jumamiller** [comment]
> @jumamiller Done

Hi, we can't pinpoint every single mistake in the entire article. It has a lot of grammatical mistakes, no transitions or flows making it hard to follow along.

I'll close this article for now, but you may resubmit a well-polished article version for the review!

Thank you!
